src/index.ts

```diff
@@ -81,6 +82,7 @@ export default function gateLogger(projectID: string, apiKey: string) {
 
             const result = await postQuery(gateURI, projectID, {
                 ...res.locals.graphqlGate,
+                requestIP: req.ip,
                 timestamp,
```

Comment:
this should be `requestIP: req.ips ? req.ips[0] : req.ip` where `req.ips` is an array of all the ip address of all the proxies the request has gone through. `req.ips[0]` is the origin ip address.

Reply:
This was also changed to `requestUuid`. TBD on what data we're storing